deployment/docker-compose/litellm_proxy_config.yaml

```diff
@@ -12,11 +12,11 @@ model_list:
     litellm_params:
       model: gpt-4o
       api_key: "os.environ/OPENAI_API_KEY"
+  # - model_name: generate-response
```

Comment:
As discussed with @amiraliemami  changed `generate-gemini-response` to `generate-response`.